app/(dashboard)/dashboard/editor/[userId]/[eventId]/page.tsx

```diff
@@ -1,204 +1,85 @@
-                        {eventInfo.label}
-                      </h2>
-                      <input
-                        type={eventInfo.type}
-                        name={eventInfo.label.toLowerCase()}
-                        placeholder={eventInfo.placeholder}
-                        className="my-2 w-full appearance-none rounded-md border-none"
-                      />
-                    </div>
-                  ))}
-              </div>
-              <div className="my-4 rounded-md border bg-background p-4 ">
-                {eventInfoList
-                  .filter((eventInfo) => eventInfo.type !== 'color')
-                  .map((eventInfo) => (
-                    <div key={eventInfo.id} className="mb-4">
-                      <label className="mb-2 block font-bold text-gray-700">
-                        {eventInfo.label}:
-                      </label>
-                      <div className="flex">
-                        <input
-                          type={eventInfo.type}
-                          name={eventInfo.label.toLowerCase()}
-                          placeholder={eventInfo.placeholder}
-                          className="w-full rounded-lg border px-4 py-2 focus:border-blue-300 focus:outline-none focus:ring"
-                        />
-                        {eventInfo.id > 6 && ( // Render delete button for newly added events
-                          <Button
-                            text="Delete"
-                            size="sm"
-                            className="ml-2 w-20 bg-red-500 text-white hover:bg-red-300"
-                            onClick={() => handleDeleteEventInfo(eventInfo.id)}
-                          />
-                          // <button
-                          //   type="button"
-                          //   onClick={() => handleDeleteEventInfo(eventInfo.id)}
-                          //   className="ml-2 rounded-lg bg-red-500 px-2 py-1 font-semibold text-white shadow-md hover:bg-red-600 focus:outline-none focus:ring focus:ring-red-300"
-                          // >
-                          //   Delete
-                          // </button>
-                        )}
-                      </div>
-                    </div>
-                  ))}
-              </div>
-
-              <div className="mb-4"></div>
-            </div>
-          </div>
+        <div className="p-4">
+          <WYSIWYGContainer
+            content={htmlState}
+            handleChange={handleWYSIWYGChange}
+          />
         </div>
       </div>
       <EditorRightColumn
         handleSave={handleSave}
-        handleAddEventInfo={handleAddEventInfo}
         showSavedMessage={showSavedMessage}
-        newEventLabel={newEventLabel}
-        setNewEventLabel={setNewEventLabel}
         userId={params.userId}
         eventId={params.eventId}
+        backgroundColor={backgroundColor || '#fffff'}
+        textColor={textColor || '#000000'}
```

Comment:
we dunt need these || OR operations right since in the useState in line 14 and 15 the default colors are already set :'0

Reply:
Nice catch!